Enable kolabd after configuration is complete.
kolabd is disabled when installed.

--- a/pykolab/setup/setup_kolabd.py
+++ b/pykolab/setup/setup_kolabd.py
@@ -62,6 +62,14 @@
             conf.cfg_parser.write(fp)
             fp.close()
 
+    if os.path.isfile('/etc/default/kolab-server'):
+        myaugeas = Augeas()
+        setting = os.path.join('/files/etc/default/kolab-server','START')
+        if not myaugeas.get(setting) == 'yes':
+            myaugeas.set(setting,'yes')
+            myaugeas.save()
+        myaugeas.close()
+
     if os.path.isfile('/bin/systemctl'):
         subprocess.call(['/bin/systemctl', 'restart', 'kolabd.service'])
     elif os.path.isfile('/sbin/service'):
